config/index.js

```diff
@@ -34,7 +34,8 @@ const config = {
     api: {
       bodySizeLimit: process.env.API_BODY_SIZE_LIMIT,
       parameterLimit: process.env.API_PARAMETER_LIMIT,
-      port: process.env.PORT
+      port: process.env.PORT,
+      weet: process.env.WEET_API
```

Comment:
Maybe something like this
```suggestion
      port: process.env.PORT
   },
   weetsApi: {
     url: process.env.WEET_API
   }
```

Reply:
changed